src/rust/engine/options/src/parse.rs

```diff
@@ -327,13 +332,19 @@ pub(crate) fn parse_dict(value: &str) -> Result<DictEdit, ParseError> {
 }
 
 pub(crate) trait Parseable: Sized + DeserializeOwned {
+    fn option_type() -> &'static str;
```

Comment:
This is just a little hack to save on boilerplate repetition in the error message generation.

Reply:
I believe you can define this as an "associated constant":

```
const OPTION_TYPE: &'static str;
```

Then implementations of the trait will set:

```
const OPTION_TYPE: &'static str = "foo";
```

Reply:
Oh, nice! Also, that made me notice that there was a bit more boilerplate to remove this way. Changed.